README.md

```diff
@@ -4,40 +4,119 @@
 [![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/fgnt/fgnt/3/master)](https://dev.azure.com/fgnt/fgnt/_build/latest?definitionId=3&branchName=master)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/fgnt/lazy_dataset/blob/master/LICENSE)
 
-Padertorch is designed to simplify the training of deep learning models written with PyTorch.
+Padertorch is designed to simplify the training of deep learning models written with [PyTorch](https://pytorch.org).
 While focusing on speech and audio processing, it is not limited to these application areas.
 
 This repository is currently under construction.
 
-The examples in contrib/examples are only working in the Paderborn NT environment
-
+[//]: <> (The examples in contrib/examples are only working in the Paderborn NT environment)
 
 # Highlights
 
-## padertorch.Trainer
-
-- Logging:
-  - The `review` of the model returns a dictionary that will be logged and visualized via 'tensorboard'. The keys define the logging type (e.g. `scalars`).
-  - As logging backend we use `TensorboardX` to generate a `tfevents` file that can be visualized from a `tensorboard`.
-- Dataset type: 
-  - lazy_dataset.Dataset, torch.utils.data.DataLoader and other iterables...
-- Validation:
-  - The `ValidationHook` runs periodically and and logs the validations results.
-- Learning rate decay with backoff:
-  - The `ValidationHook` has also parameters to do a learning rate with backoff.
-- Test run: 
-  - The trainer has a test run function to train the model for few iterations and test if
+- **Fast prototyping**: The trainer and models are all compatible with [sacred](https://github.com/IDSIA/sacred) and easily allow hyperparameter changes over the command line. Check out the [configurable](padertorch/configurable.py) module and the [examples](padertorch/contrib/examples) for how it works.
+- **Easily extensible**: Write your own network modules and models based on `padertorch.Module` and `padertorch.Model`.
+- **Seamless integration**: You provide your own data and model. We provide the trainer to wrap around your model and data. One simple command starts the training: `padertorch.Trainer.train`.
```

Comment:
I think this gives the wrong impression that train works without initialization. Perhaps just write that calling train starts the training?